### PR TITLE
Dang/add port id into msg register interchain account response

### PR DESCRIPTION
**Pull request by GNaD13**
## Description

Add generated portID to message MsgRegisterInterchainAccountResponse

closes: #2724

### Commit Message / Changelog Entry

```bash
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/10-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: proto/ibc/applications/interchain_accounts/controller/v1/tx.proto [hunk 1]
```diff
@@ -28,6 +28,7 @@ message MsgRegisterInterchainAccount {
 // MsgRegisterInterchainAccountResponse defines the response for Msg/RegisterAccount
 message MsgRegisterInterchainAccountResponse {
   string channel_id = 1 [(gogoproto.moretags) = "yaml:\"channel_id\""];
+  string port_id = 2 [(gogoproto.moretags) = "yaml:\"port_id\""];
```
Comment (damiannolan) [inline proto/ibc/applications/interchain_accounts/controller/v1/tx.proto:31]: Could you run `make proto-all` or `make proto-format`? Looks like the alignment is off, maybe you just used the `proto-gen` target.

**Review by damiannolan — COMMENTED** [review]
Thanks for picking this up! Left a suggestion on proto formatting. 

Could you add a changelog entry under improvements?

**Review by crodriguezvega — APPROVED** [review]
Thank you, @GNaD13!

Can you also please update the `MsgRegisterInterchainAccountResponse` message in the docs [here](https://github.com/cosmos/ibc-go/blob/main/docs/apps/interchain-accounts/messages.md#msgregisterinterchainaccount)?

File: proto/ibc/applications/interchain_accounts/controller/v1/tx.proto (hunk 1 above)
Comment (GNaD13) [reply proto/ibc/applications/interchain_accounts/controller/v1/tx.proto]: Hi sir @damiannolan , thanks for your reviewed. I've fixed it

**Comment by GNaD13** [comment]
> Thank you, @GNaD13!
> 
> Can you also please update the `MsgRegisterInterchainAccountResponse` message in the docs [here](https://github.com/cosmos/ibc-go/blob/main/docs/apps/interchain-accounts/messages.md#msgregisterinterchainaccount)?

Hi sir @crodriguezvega , Thanks for your reviewed, I've updated it

File: docs/apps/interchain-accounts/messages.md [hunk 2]
```diff
@@ -29,10 +29,11 @@ If the `Version` string is omitted, the controller submodule will construct a de
 ```go
 type MsgRegisterInterchainAccountResponse struct {
   ChannelID string
+  PortId string
 }
 ```
 
-The `ChannelID` is returned in the message response.
+The `ChannelID` and `PortID` is returned in the message response.
```
Comment (crodriguezvega) [inline docs/apps/interchain-accounts/messages.md:36]: Super nit :) 

```suggestion
The `ChannelID` and `PortID` are returned in the message response.
```

**Review by crodriguezvega — APPROVED** [review]
Thanks for update, @GNaD13! I just have one little nit. :)

File: docs/apps/interchain-accounts/messages.md (hunk 2 above)
Comment (GNaD13) [reply docs/apps/interchain-accounts/messages.md]: oops, thank you sir :3

**Review by damiannolan — APPROVED** [review]
🙏

**Pull request merged**